.eslintignore

```diff
@@ -1 +1,5 @@
 addon/StudyUtils.jsm
+dist
+OLD
+package-lock.json
+!.eslintrc.js
```

Comment:
This stealthy negation makes sure we _do_ lint the .eslintrc.js file (since files prefixed with a period are ignored by default).